### PR TITLE
CINAPI-14 Added a function for getting UTC offset from a date

### PR DESCRIPTION
**Pull request by andrevez**

### CODE REVIEW

File: src/getUTCOffsetFromDate.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import dayjs from 'dayjs';
+import {isString} from 'lodash';
```
Comment (slavik-chapelskyi) [inline src/getUTCOffsetFromDate.ts]: This package do not depend on lodash. 
Please use typeof === 'string' alternative

File: src/getUTCOffsetFromDate.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+export default (date: string | Date): string => {
+  if (isString(date)) {
```
Comment (slavik-chapelskyi) [inline src/getUTCOffsetFromDate.ts]: Better to always pass to dayjs. 
Eg. if not full date strinbg will be provided, like `2019-01-25T00:00:00`

File: src/getUTCOffsetFromDate.ts (hunk 1 above)
Comment (andrevez) [reply src/getUTCOffsetFromDate.ts]: thx, didn't notice

File: src/getUTCOffsetFromDate.ts (hunk 2 above)
Comment (andrevez) [reply src/getUTCOffsetFromDate.ts]: Not sure I can do that, `dayjs` just ignores the UTC offset from the string. Adding more code just to handle the `date string` without offset makes the function more complicated but the result will be the same with or without it.

**Review by hmelenok — APPROVED** [review]

**Review by Maksym-Haiovets — APPROVED** [review]

**Review by KnupMan — APPROVED** [review]

**Pull request merged**